src/scripts/setup/babel.js

```diff
@@ -0,0 +1,21 @@
+const { fromRoot, hasFile, writeFileToRoot } = require('../../utils')
+
+const babelConfig = fromRoot('./.babelrc')
+
+const setupBabel = () => {
+  if (hasFile('./.babelrc')) {
+    console.log(`.babelrc already exists at ${babelConfig}`)
+    console.log('Cancelling Babel setup')
+    process.exit(0)
+  }
+
+  const content = '{ "presets": ["@helpscout/zero/babel"] }'
+
+  console.log(`Generating ${babelConfig}...`)
+  writeFileToRoot('./.babelrc', content)
+
+  console.log()
```

Comment:
Is this empty `console.log` intentional?

Reply:
I see a few, so maybe they are, curious about the intention!

Reply:
@tinkertrain That's just to leave a new line `\n` in the log :)

So it would be

```
one

two
```

vs

```
one
two
```